### DISCUSSION TITLE
Run time question

### DISCUSSION THREAD

**Discussion by laurenjonas1** [bio-miga/miga#182] [General]
I currently have three jobs running against NCBI Prok that have been running for 27 days. Two are against the TypeMat db and have been running for 18 days. Is this normal or did something go wrong? The only information I can see is that the contigs are currently being processed for cds.

Any advice?

**Comment by MohammadKaddoura** [comment]
I'm having the same issue. I have a job that's been on NCBI Prok for more than a month and another on TypeMat thats been for almost 2 weeks.

**Comment by MicoNick** [comment]
I am having the same problem. I have a job that is in NCBI Prok and another in TypeMat that have been in NCBI Prok for over 11 days without any updates.
